package.json

```diff
@@ -18,6 +18,7 @@
     "compile:js": "sgmf-scripts --compile js",
     "transpile": "babel ./src/cartridges -d ./cartridges --copy-files --no-copy-ignored --extensions .js,.test.js,.snap",
     "build": "npm run transpile && npm run compile:js && npm run uploadCartridge",
+    "removeAndBuild": "rm -rf cartridges && npm run build",
```

Comment:
The "rm" command will not work on windows. We could use babel flag " --delete-dir-on-start" to clean the output directory before babel runs.

Reply:
Hey, aparently that command creates nested cartridges folder. 
I use rimraf instead, I see it works on windows as well.